src/components/NavBar/NavDropdown.tsx

```diff
@@ -23,13 +24,14 @@ export const NavDropdown = ({
   paddingTop,
   children,
 }: NavDropdownProps) => {
+  const isMobile = useIsMobile()
```

Comment:
There seem to be three ways for styles to be decided:
- `centerHorizontally`, etc -- passed in as props
- `isMobile` -- from the hook
- `sprinkles` -- breakpoints in the css file

I think consolidating these more would help the understanding of the file a lot